x-pack/plugins/cases/server/routes/api/utils.test.ts

```diff
@@ -54,4 +54,55 @@ describe('Utils', () => {
       expect(res.headers).toEqual({});
     });
   });
+  describe('getCaseUrl', () => {
```

Comment:
I think we should add a test when you have basePath, just to make sure it is working as expected